tests/fixture/e2e/helpers.go

```diff
@@ -146,7 +146,10 @@ func AddEphemeralNode(network testnet.Network, flags testnet.FlagsMap) testnet.N
 
 // Wait for the given node to report healthy.
 func WaitForHealthy(node testnet.Node) {
-	require.NoError(ginkgo.GinkgoT(), testnet.WaitForHealthy(DefaultContext(), node))
+	// Need to use explicit context (vs DefaultContext()) to support use with DeferCleanup
+	ctx, cancel := context.WithTimeout(context.Background(), DefaultTimeout)
+	defer cancel()
+	require.NoError(ginkgo.GinkgoT(), testnet.WaitForHealthy(ctx, node))
```

Comment:
Would it make sense for us to pass a `GinkgoTInterface` from the caller here as well? I'm not that familiar on Ginkgo but something seems weird to me about this line because I believe the caller is generally creating creating its own instance of `GinkgoT`.

Reply:
offline answer: ginkgo returns the same thing regardless of where it's called so there's no difference